docs/documentation/head/connect.md

```diff
@@ -424,6 +424,15 @@ Connection conn = DriverManager.getConnection(url);
 
 	Put the connection in read-only mode
 
+* **readOnlyMode** = String
+	
+	Controls the behavior when a connection is set to be read only, one of 'ignore', 'transaction', or 'always'. 
+	When 'ignore', setting readOnly has no effect. 
+	When 'transaction' setting readOnly to 'true' will cause transactions to BEGIN READ ONLY if autocommit is 'false'.
```

Comment:
can you change this to either "issue" or "emit" BEGIN READ ONLY

Reply:
@davecramer I think this is just a copy of the original PR: https://github.com/pgjdbc/pgjdbc/pull/1252/files#diff-9c1bd193fba088d6998a42dbc12c7b53R453-R459

Would be good to merge at least this, because right now there is no documentation at all about `readOnlyMode`

Reply:
@mkurz It appears the original was merged in already